src/codegen/names.jl

```diff
@@ -19,6 +20,7 @@ const JULIA_RESERVED_KEYWORDS = Set{String}([
     "end",
     "Enum",
     "export",
+    "Expr",
```

Comment:
This might be too defensive, but I think I saw a clash of some sort when defining a struct named Expr before...